web-common/src/main/java/edu/unc/lib/boxc/web/common/utils/AnalyticsTrackerUtil.java

```diff
@@ -107,6 +116,43 @@ private void trackEvent(AnalyticsUserData userData, String category, String acti
         trackerThread.start();
     }
 
+    private MatomoRequest buildMatomoRequest(String url, AnalyticsUserData userData, String parentCollection, String label) throws UnsupportedEncodingException {
+        return MatomoRequest.builder()
+                .siteId(MATOMO_SITE_ID)
+                .visitorId(userData.uid)
+                .actionUrl(urlEncode(url))
+                .actionName(urlEncode(MATOMO_ACTION))
```

Comment:
action name can be lots of things but apparently it's best practice to make it equal eventCategory + eventAction like "Help / Feedback" where Help is the category and Feedback is the action. I've updated it to be that now.

action url and download url are the same value, but adding the download url tells matomo that this is a download. action url is just the url where the action occurred.

i'm getting rid of the download url param since we are going with events